kubernetes-model/src/main/resources/schema/validation-schema.json

```diff
@@ -16963,44 +16963,9 @@
     },
     "deploymentstrategy": {
       "properties": {
```

Comment:
should we be concerned about all the deleted config below?

Reply:
Hmmm, weird. Given the use we make of the lib I'd say "if nothing seems to break in the end, don't bother" but I'll spend 5mins trying to understand how this happened

Reply:
It is weird indeed, as we depend on a given sha of the kubernetes/openshift APIs (and the `.lock` file did not seem to change in that regard). Neither did the upstream repo in the last week.